manifest.md

```diff
@@ -42,6 +42,12 @@ Unlike the [image index](image-index.md), which contains information about a set
 
         If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
 
+    To set an effectively NULL or SCRATCH config and maintain portability the following is considered GUIDANCE.
+    While an empty blob (`size` of 0) may be preferable, practice has shown that not to be ubiquitiously supported.
```

Comment:
> practice has shown that not to be ubiquitiously supported.

This might not be a problem, because implementations do not need to unmarshal this "JSON" object (and the unmarshallized object is simply useless)

Reply:
Practice has shown zero size blobs to fail.

Reply:
see #1025